libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -522,6 +522,9 @@ class YulUtilFunctions
 	/// Signature: (address) -> mpos
 	std::string externalCodeFunction();
 
+	/// @return Yul Code to check if two external functions pointers are equal or not
+	std::string checkExternalFunctionsEqual();
```

Comment:
I guess the canonical naming scheme for these would mean this should rather be called something like
```suggestion
	std::string externalFunctionPointerEqualFunction();
```
Not that that's overly nice, but we should at least stick to the schema of ``whateverFunction``